CHANGELOG.md

```diff
@@ -5,8 +5,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
-===================
+## [v23.0.0] - 2023-05-15
+
+### Notes
+
+- This release requires Python >=3.11. Lower versions may also work but are untested
+- The repository has been refactored to use `pyproject.toml` as the source of truth for dependency requirements
+- Added `.python-version` file to keep track of supported Python version
+
+### Added
+- Github Actions for CI/CD pipelines
```

Comment:
we probably don't need to create the section and one PR is pending that we should merge and add here.

Reply:
I think he has a parent bullet for CI/CD pipelines ... and the items below are subbullets to it.

Reply:
@mangalaman93 they are all related to the same solution (adding ci/cd pipelines).  It is more clear this way